lib/transmissions.js

```diff
@@ -81,3 +91,97 @@ module.exports = function(client) {
   };
 
 };
+
+function formatPayload(transmission) {
+  // format all the original recipients to be in the object format
+  transmission.recipients = _.map(transmission.recipients, (recipient) => {
+    recipient.address = addressToObject(recipient.address);
+
+    return recipient;
+  });
+
+  // add the CC headers
+  if (_.has(transmission, 'cc') && _.isArray(transmission.cc)) {
+    addCCHeader(transmission);
+  }
+
+  addListToRecipients(transmission, 'cc');
+  addListToRecipients(transmission, 'bcc');
+
+  return transmission;
+}
+
+function addListToRecipients(transmission, listName) {
+  if (_.has(transmission, listName) && _.isArray(transmission[listName])) {
+    const headerTo = generateHeaderTo(transmission.recipients);
+    const list = transmission[listName];
+
+    transmission.recipients = transmission.recipients.concat(_.map(list, (recipient) => {
+      recipient.address = addressToObject(recipient.address);
+
+      recipient.address.header_to = headerTo;
+
+      // remove name from address - name is only put in the header for cc and not at all for bcc
+      if (_.has(recipient.address, 'name')) {
+        delete recipient.address.name;
+      }
+
+      return recipient;
+    }));
+
+    delete transmission[listName];
+  }
+}
+
+function addCCHeader(transmission) {
+  // default the headers to a blank object
+  transmission.content.headers = transmission.content.headers || {};
+
+  transmission.content.headers.CC = _.map(transmission.cc, (ccRecipient) => addressToString(ccRecipient.address)).join(', ');
+}
+
+function generateHeaderTo(recipients) {
+  const originalRecipients = _.filter(recipients, (recipient) => !_.has(recipient.address, 'header_to'));
```

Comment:
What does this line do? I don't see any examples where `header_to` is ever defined so I forget why this scenario would arise.